integration-test/src/main/java/com/sequenceiq/it/cloudbreak/util/wait/FlowUtil.java

```diff
@@ -53,8 +53,12 @@ private void waitForFlow(FlowPublicEndpoint flowEndpoint, String crn, String flo
         boolean flowFailed = false;
 
         int retryCount = 0;
+        long pollingInt = pollingInterval;
```

Comment:
`pollingInterval`, `pollingInt`? Out of good words? Can you please do the following instead?

```java
long pollingInterval = getPollingInterval(runningParameter);

...

    private long getPollingInterval(RunningParameter runningParameter) {
        if (runningParameter.getPollingInterval() != null) {
            return runningParameter.getPollingInterval().toMillis();
        }
        return pollingInterval;
    }
```

Reply:
I have noticed the code duplication, so I will merge them